WorkerApi/ArmoniK.DevelopmentKit.WorkerApi/Services/ComputerService.cs

```diff
@@ -156,15 +158,16 @@ public override async Task<Output> Process(ITaskHandler taskHandler)
         Logger.LogError(ex,
                         "WorkerAPIException failure while executing task");
 
-        throw;
+        throw new RpcException(new Status(StatusCode.Aborted,
+                                          ex.Message + ex.StackTrace));
```

Comment:
Shouldn't be there a line return between the message and the stacktrace?

Reply:
done

Reply:
It's present in the ex.StackTrace but just in case I added Environement.NewLine